vendor/github.com/hashicorp/serf/serf/keymanager.go

```diff
@@ -189,4 +189,4 @@ func (k *KeyManager) ListKeysWithOptions(opts *KeyRequestOptions) (*KeyResponse,
 	defer k.l.RUnlock()
 
 	return k.handleKeyRequest("", listKeysQuery, opts)
-}
\ No newline at end of file
+}
```

Comment:
Mind reverting these vendor changes? I think govendor hashes vendored packages, so even single byte changes might throw something off.

Reply:
oops yeah that must've been an accidental overreaching gofmt

Reply:
fixed